lib/default-theme/Layout.vue

```diff
@@ -111,6 +112,8 @@ export default {
     this.$watch('$page', updateMeta)
     updateMeta()
 
+    window.addEventListener('scroll', this.onScroll)
```

Comment:
Should this live in `created()` hook? If the user scrolled after `mounted()` but before `created()`, will `this.onScroll` be available?

Reply:
You ask a contributor to access `window ` at `created` hook? Are you kidding me?

Reply:
Emmm... In addtion, what is 

> after `mounted()` but before `created()`

Reply:
@ycmjason Pay attention to your wording. If I don't read your comment, then you will be conveying a very low-level error in a vue official project. Is this reasonable? 

BTW, this is not your project, please take a good look at Vue before you speak anything, or I'll think you're just making trouble.

Reply:
@ulivz please pay attention to yours as well. You are all contributors. You all can make mistakes and can have different opinions. Point out problems with neutral language, don't bring your ego into this.

Reply:
@ulivz please pay attention to yours as well. You are all contributors. You all can make mistakes and can have different opinions. Point out problems with neutral language, don't bring your ego into this.